homeassistant/components/image_processing/opencv.py

```diff
@@ -0,0 +1,121 @@
+"""
+Component that performs OpenCV processes on images.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/image_processing.opencv/
+"""
+import asyncio
+import logging
+
+from homeassistant.components.opencv import (
+    ATTR_MATCHES,
+    CLASSIFIER_GROUP_CONFIG,
+    CONF_CLASSIFIER,
+    CONF_ENTITY_ID,
+    CONF_NAME,
+    DOMAIN as OPENCV_DOMAIN,
+    process_image,
+)
+from homeassistant.components.image_processing import (
```

Comment:
Sort imports alphabetically. This should go above `homeassistant.components.opencv`.